libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -250,11 +251,25 @@ class YulUtilFunctions
 	/// Return value: pointer
 	std::string allocationFunction();
 
-	/// @returns the name of a function that allocates a memory array.
+	/// @returns the name of a function that zeroes an array
+	/// where the base has simple zero value in memory.
+	/// signature: (headPointer, allocatedSize) ->
```

Comment:
Would it be easier to use and understand if the parameters would be `(dataStart, dataSizeInBytes)` ?

Reply:
So that the caller would need to decide what to pass depending on `dynamic`?

Reply:
yes